src/main/java/at/hannibal2/skyhanni/features/misc/reminders/ReminderManager.kt

```diff
@@ -42,7 +42,7 @@ object ReminderManager {
     private fun newId(): String {
         var id: String
         do {
-            id = Base62.encode((Math.random() * Int.MAX_VALUE).toInt())
+            id = UUID.randomUUID().toString()
         } while (storage.containsKey(id))
```

Comment:
Please remove this storage check here.
Even with one million reminders in the list, the probability of a collision when creating a new random UUID is around `9.4 x 10^-26`.
Instead, we could remove the whole function and replace it with https://github.com/hannibal002/SkyHanni/blob/40faf50e3f109d0028e67b6dd916ff79f2cb5604/src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt#L386

Reply:
I've maybe worded that a bit wrong. Fixed it here https://github.com/hannibal002/SkyHanni/pull/1708/commits/3101b6883fe95b4c6dbf71776083e63794c7b0b2